plugins/servicenow/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: ["insightconnect"]
 name: servicenow
 title: ServiceNow
 description: ServiceNow is a tool for managing incidents and configuration management. Using the ServiceNow plugin for Rapid7 InsightConnect, users can manage all aspects of incidents including creation, search, updates, as well as monitor them for changes
-version: 7.2.0
+version: 7.3.0
```

Comment:
Need to add in connection version also?

Reply:
@dlaverty-r7 I added the connection version.